### ISSUE TITLE
-Wsign-compare warnings

### ISSUE THREAD

**Issue by adumasphi** [persalys/persalys#11]
In GitLab by @schueller on Oct 13, 2016, 09:58

/home/schueller/projects/otgui/lib/src/base/MonteCarloAnalysis.cxx:118:23: warning: comparison of integers of different signs: 'clock_t' (aka 'long') and 'unsigned long' [-Wsign-compare]
     &&  (elapsedTime < getMaximumElapsedTime()*CLOCKS_PER_SEC))
          ~~~~~~~~~~~ ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[ 16%] Building CXX object lib/src/base/CMakeFiles/otguibase.dir/LimitStateImplementation.cxx.o                                                                                                                                             
[ 16%] Building CXX object lib/src/base/CMakeFiles/otguibase.dir/LimitState.cxx.o
/home/schueller/projects/otgui/lib/src/base/SobolAnalysis.cxx:97:23: warning: comparison of integers of different signs: 'clock_t' (aka 'long') and 'unsigned long' [-Wsign-compare]
     &&  (elapsedTime < getMaximumElapsedTime()*CLOCKS_PER_SEC))

**Comment by adumasphi** [comment]
In GitLab by @schueller on Oct 13, 2016, 15:25

Mentioned in commit 85eb8accbc97c4884db2af2fe8391902031bd945

**Comment by adumasphi** [comment]
In GitLab by @schueller on Oct 13, 2016, 15:25

Mentioned in merge request !40

**Comment by adumasphi** [comment]
In GitLab by @schueller on Oct 13, 2016, 15:55

Status changed to closed by commit 85eb8accbc97c4884db2af2fe8391902031bd945

**Comment by adumasphi** [comment]
In GitLab by @schueller on Nov 2, 2016, 15:22

Mentioned in commit 54f8b0845c95e19ecf4e2940b945df5ad3b8f5d0
